pkg/staticnetworkconfig/generator.go

```diff
@@ -180,7 +180,9 @@ func (s *StaticNetworkConfigGenerator) validateInterfaceNamesExistence(macInterf
 		connectionSection := cfg.Section("connection")
 		interfaceName, err := connectionSection.GetKey("interface-name")
 		if err != nil {
-			return errors.Wrapf(err, "failed to get interface-name for file %s", nc.FilePath)
+			// When the id field is provided the interface-name will not be present
+			matched = true
```

Comment:
There is no match here.

Reply:
matched setting has been removed

Reply:
I still see it in this commit... I know you removed it eventually but this commit has to be fixed as well.

Reply:
Got it. Updated commit to remove that line.